packages/ui/src/theme/index.ts

```diff
@@ -1,6 +1,5 @@
+import { consoleDarkTheme, consoleLightTheme } from '@scaleway/themes'
```

Comment:
@chambo-e From now on it doesn't make much sense as we only have 2 themes available. In a soon future, design wants to make 2 themes per product (documentation, website, etc). All those themes won't be handle in Scaleway UI, so in order to avoid pollution we need to put themes in another package.

This way, documentation could install Scaleway UI, import documentation light and dark theme and inject it into their ThemeProvider.